src/src/App.svelte

```diff
@@ -49,9 +52,12 @@
 							console.error(`Failed to fetch movie data for ID ${entity.movieId}:`, movieResponse.statusText);
 							return null;
 						}
-
 						const movieData = await movieResponse.json();
-						return movieData.data;
+						const movie: Movie = movieData.data;
+
+						console.log('Fetched movie:', movie);
```

Comment:
```suggestion
```

Reply:
this should also be removed with the new pull request !